cli/util/uploader.go

```diff
@@ -206,20 +206,18 @@ func fetchLatestMissingBlockIndex(ctx context.Context, p *pool.Pool, containerID
 			wg.Add(1)
 			go func(i, startIndex, endIndex int) {
 				defer wg.Done()
-
-				prm := client.PrmObjectSearch{}
-				filters := object.NewSearchFilters()
-				filters.AddFilter(attributeKey, fmt.Sprintf("%d", startIndex), object.MatchNumGE)
-				filters.AddFilter(attributeKey, fmt.Sprintf("%d", endIndex), object.MatchNumLT)
-				prm.SetFilters(filters)
+				objCh := searchObjects(ctx, p, containerID, acc, attributeKey, uint(startIndex), uint(endIndex), 1, errCh)
```

Comment:
@AnnaShaleva are you okay with such reuse of `searchObjects`? Because of per batch checking, we can not pass the full range.

Reply:
I see the problem. Then no, I'm not OK, let's keep the old code for `fetchLatestMissingBlockIndex` because otherwise there's additional unnecessary overheads on `objCh` management.

Reply:
@AliceInHunterland, let's fix it and then I'll review one more time.